script.js

```diff
@@ -0,0 +1,65 @@
+let bookList = [];
+const awesomeBook = document.getElementById('awesome-book');
+const bttn = document.getElementById('bttn');
+awesomeBook.style.display = 'none';
+
+function displayBooks(list) {
+  let tr = '';
+  let sn = 1;
+  list.forEach((item) => {
+    const tr1 = `<tr>
+    <td>${sn}</td>
+    <td>${item.title}</td>
+    <td>${item.author}</td>
+    <td> <button type ="button" class = "remove-book" data-bookId="${(sn - 1)}">Remove</button></td>
+    </tr>`;
+    tr += tr1;
+    sn += 1;
+  });
+  document.getElementById('tbody').innerHTML = tr;
+}
+
+function getLocalStorageData() {
+  const data = JSON.parse(localStorage.getItem('bookdata'));
+  if (data != null && data.length > 0) {
+    awesomeBook.style.display = 'block';
+    displayBooks(data);
+  } else {
+    awesomeBook.style.display = 'none';
+  }
+}
+
+bttn.onclick = function () {
```

Comment:
- [ ] Kindly you should use ES6 instead of using function manually. to convert that to arrow function  it should be like `bttn.onclick = () => {..........}`